### ISSUE TITLE
Docx file created by LibreOffice returns Zip mime type

### ISSUE THREAD

**Issue by RangerMak** [j256/simplemagic#38]
I think its because LibreOffice (OpenOffice) put files into docx-archive in different order than MS Word.
So file have another signature and detects as simple zip-archive.

Example file is attached.
[DocxByLibreOffice.docx](https://github.com/j256/simplemagic/files/1498249/DocxByLibreOffice.docx)

**Comment by j256** [comment]
That's correct because it _is_ a zip file.  For me to detect it as something else means that I have to open it up and process the file contents which I'm less interested in doing.

**Comment by fjtorres** [comment]
@j256 Did you add something to detect this case as a DOCX file? I found similar problem and I tried several ways to detect it as a DOCX file but just "Tika core" library was able to detect this case correctly.

```
Tika tika = new Tika();
String mimeType = tika.detect(filePath);
// output mime type: application/vnd.openxmlformats-officedocument.wordprocessingml.document
```
